README.md

```diff
@@ -101,8 +101,8 @@ This will override any python version specifications in `pyproject.toml` and `.p
 - name: Install the latest version of uv and set the python version to 3.13t
   uses: astral-sh/setup-uv@v6
   with:
-    python-version: 3.13t
-- run: uv pip install --python=3.13t pip
+    python-version: 3.13
+- run: uv pip install --python=3.13 pip
```

Comment:
the "t" stands for the free-threaded version, so it wasn't a typo as you assumed